### PR TITLE
removed dynamic price on publish, liquidity button and trade tab

### PR DESCRIPTION
**Pull request by EnzoVezzaro**
Fixes #.

Changes proposed in this PR:

- Removed trade tab
- Removed dynamic pricing schema in the publish flow
- Removed 'Add liquidity' option

### CODE REVIEW

File: src/components/Asset/AssetActions/Pool/index.tsx [hunk 1]
```diff
@@ -12,43 +11,33 @@ import { usePool } from '@context/Pool'
...
-            {hasUserAddedLiquidity && (
+          {hasUserAddedLiquidity && (
+            <div className={stylesActions.actions}>
               <Button
+                style="primary"
                 size="small"
                 onClick={() => setShowRemove(true)}
                 disabled={!isAssetNetwork}
               >
                 Remove
```
[the remark this answers is not in the document]
Comment (EnzoVezzaro) [reply src/components/Asset/AssetActions/Pool/index.tsx]: Sure 👍🏽

File: src/components/Publish/Pricing/index.tsx [hunk 2]
```diff
@@ -67,12 +67,6 @@ export default function PricingFields(): ReactElement {
           content: <Fixed content={content.create.fixed} />
         }
       : undefined,
-    appConfig.allowDynamicPricing === 'true'
```
Comment (kremalicious) [inline src/components/Publish/Pricing/index.tsx:70]: Can also remove both effects in lines 32-61 from this file

Comment (EnzoVezzaro) [reply src/components/Publish/Pricing/index.tsx:70]: 🚀

**Comment by EnzoVezzaro** [comment]
> image fixed price validation seems busted

ok, yes. this last commit broke it

**Comment by bogdanfazakas** [comment]
> UPDATE: for some reason removing [these](https://github.com/oceanprotocol/market/pull/1593#discussion_r919002567) lines causes the pricing to not being validated correctly. I reverted these last changes while I'm trying to solve it

Validation seem to work on my end for fre but actual publish doesn't. Maybe it's related to your comment @EnzoVezzaro 
Screenshot 2022-07-12 at 17 32 46

**Comment by EnzoVezzaro** [comment]
> > UPDATE: for some reason removing [these](https://github.com/oceanprotocol/market/pull/1593#discussion_r919002567) lines causes the pricing to not being validated correctly. I reverted these last changes while I'm trying to solve it
> 
> Validation seem to work on my end for fre but actual publish doesn't. Maybe it's related to your comment @EnzoVezzaro Screenshot 2022-07-12 at 17 32 46

Yeah exactly. I had some leftovers from the default values. All this logic makes it kind of hard to move around, but now it should be at least be ready to go. I tested both free and fixed pricing. Both successful.

**Review by mihaisc — APPROVED** [review]

**Comment by AnaLoznianu** [comment]
LGTM

**Pull request merged**